### ISSUE TITLE
Can't use on colab

### ISSUE THREAD

**Issue by tenajima** [tenajima/nullpom#13]
- Because of pandas version.
![image](https://user-images.githubusercontent.com/19265782/111013316-3df13e80-83e2-11eb-95a8-2c3e4dac05cd.png)

**Comment by tenajima** [comment]
log
```
Collecting threadpoolctl>=2.0.0
  Downloading https://files.pythonhosted.org/packages/f7/12/ec3f2e203afa394a149911729357aa48affc59c20e2c1c8297a60f33f133/threadpoolctl-2.1.0-py3-none-any.whl
Requirement already satisfied: scipy>=0.19.1 in /usr/local/lib/python3.7/dist-packages (from scikit-learn<0.25.0,>=0.24.1->nullpom) (1.4.1)
Requirement already satisfied: pytz>=2017.3 in /usr/local/lib/python3.7/dist-packages (from pandas<2.0.0,>=1.2.3->nullpom) (2018.9)
Requirement already satisfied: wheel in /usr/local/lib/python3.7/dist-packages (from lightgbm<4.0.0,>=3.1.1->nullpom) (0.36.2)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.7/dist-packages (from python-dateutil>=2.1->matplotlib<4.0.0,>=3.3.4->nullpom) (1.15.0)
ERROR: google-colab 1.0.0 has requirement pandas~=1.1.0; python_version >= "3.0", but you'll have pandas 1.2.3 which is incompatible.
ERROR: albumentations 0.1.12 has requirement imgaug<0.2.7,>=0.2.5, but you'll have imgaug 0.2.9 which is incompatible.
Installing collected packages: matplotlib, threadpoolctl, scikit-learn, pandas, lightgbm, nullpom
  Found existing installation: matplotlib 3.2.2
    Uninstalling matplotlib-3.2.2:
      Successfully uninstalled matplotlib-3.2.2
  Found existing installation: scikit-learn 0.22.2.post1
    Uninstalling scikit-learn-0.22.2.post1:
      Successfully uninstalled scikit-learn-0.22.2.post1
  Found existing installation: pandas 1.1.5
    Uninstalling pandas-1.1.5:
      Successfully uninstalled pandas-1.1.5
  Found existing installation: lightgbm 2.2.3
    Uninstalling lightgbm-2.2.3:
      Successfully uninstalled lightgbm-2.2.3
Successfully installed lightgbm-3.1.1 matplotlib-3.3.4 nullpom-0.0.5 pandas-1.2.3 scikit-learn-0.24.1 threadpoolctl-2.1.0
WARNING: The following packages were previously imported in this runtime:
  [matplotlib,mpl_toolkits,pandas]
You must restart the runtime in order to use newly installed versions.
```

**Comment by tenajima** [comment]
Fixed by #14
